packages/protobuf/src/wkt/index.ts

```diff
@@ -16,11 +16,13 @@ export * from "./timestamp.js";
 export * from "./any.js";
 export * from "./wrappers.js";
 export * from "./gen/google/protobuf/api_pb.js";
+export * from "./gen/google/protobuf/cpp_features_pb.js";
 export * from "./gen/google/protobuf/any_pb.js";
 export * from "./gen/google/protobuf/descriptor_pb.js";
 export * from "./gen/google/protobuf/duration_pb.js";
 export * from "./gen/google/protobuf/empty_pb.js";
 export * from "./gen/google/protobuf/field_mask_pb.js";
+export * from "./gen/google/protobuf/java_features_pb.js";
```

Comment:
[Release 26.1](https://github.com/protocolbuffers/protobuf/releases/tag/v26.1) adds these two new Protobuf files. They are relevant for compilers and C++ / Java runtimes.

Reply:
The only use case I can think of in (the context of protobuf-es) is to manipulate these features in file descriptors. Should this be necessary, it's trivial to generate the files explicitly. 

Removed them in https://github.com/bufbuild/protobuf-es/pull/793/commits/8e5f2f633104dfbbe0d46b3c34c726518cde7a21.